packages/styles/src/mixins/_tokens.scss

```diff
@@ -0,0 +1,12 @@
+@use './../tokens/temp/components';
+@use 'sass:map';
+
+@mixin from($list) {
+  @each $key, $value in $list {
+    --#{$key}: #{$value};
+  }
+}
+
+@function component($name) {
+  @return #{map.get(components.$post-interactives, $name)};
```

Comment:
This is fine for testing, but it will not work as soon as you like to get other component tokens (not from the hardcoded map `$post-interactives`).

Reply:
Yes, I think we need to either flatten the components into one map or export component-button.scss with button only tokens from the tokens package.

Reply:
@oliverschuerch I've removed the component token implementation again since we need to make changes to how component tokens are rendered.